### DISCUSSION TITLE
Is it possible to create custom file naming presets using the Labs QR code maker?

### DISCUSSION THREAD

**Discussion by Tori456** [gopro/labs#764] [Q&A]
Hi! 
Does anyone know if it would be possible to make custom file naming conventions set up via QR code, like for example: red_1_(clip number)? It would be so useful! thank you in advance, so far all the advice I have got from this forum has been so helpful and I really appreciate it.
Thank you :)

**Comment by dnewman-gpsw** [comment]
not as a preset, but you can have custom file names.  https://gopro.github.io/labs/control/basename/

**Reply by Tori456** [reply]
Thank you this has worked perfectly :)
